src/lib/getImportDeclaration.ts

```diff
@@ -0,0 +1,33 @@
+import { AST_NODE_TYPES } from "@typescript-eslint/utils";
+import { ParserServices } from "@typescript-eslint/utils";
+import { ImportDeclaration } from "@typescript-eslint/types/dist/ast-spec";
+import { Symbol } from "typescript";
+import { JSXOpeningElement } from "@typescript-eslint/utils/node_modules/@typescript-eslint/types/dist/ast-spec";
+
+export function getImportDeclarationOfJSX(
```

Comment:
`isChakraElement` should use `getImportDeclarationOfJSX`, but it's out of scope of this PR.